depositapp/android/app/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,34 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.example.depositapp">
```

Comment:
Let's make sure we get the right package names from the beginning.
Change this to io.empowerchain.depositapp

Also make sure you find any other places this needs to be changed.